loops-and-arrays.js

```diff
@@ -23,13 +26,21 @@ const names = ["Henk", "Piet", "Fred", "Joop"];
 
 const numbers = [2, 4, 5, 29, 38];
 
+console.log(numbers);
+
 // Verwachtte uitkomsten:
 // Vóór het script zie je de originele waardes:
 // console.log(numbers) geeft: [2, 4, 5, 29, 38];
 // Na jouw script zie je de aangepaste waardes:
 // console.log(numbers) geeft: [ 4, 8, 15, 87, 76 ];
 
-
+for(let i = 0; i < numbers.length; i++)   {
+    if(numbers[i] % 2 == 0)  {
+        console.log(numbers[i] * 2);
+    } else  {
+        console.log(numbers[i] * 3);
```

Comment:
Ook hier worden de aangepaste waardes alleen in de log getoond en niet aangepast in de array.